exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/sklearn/sklearn_predict_abalone.ipynb

```diff
@@ -50,17 +50,16 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    bfs_model_path = get_udf_bucket_path(sb_config) + '/abalone_svm_model.pkl'
</pre><p>Maybe use Pathlib to construct path</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
OK. os.path.join will be a better choice, as it doesn't require creating an object.

Reply:
So, maybe get_udf_bucket_path should actually return a path object

Reply:
I don't mind, but again, let's log it as a change for the next release of the notebook-connector.

Reply:
agree

Reply:
Created a [new ticket](https://github.com/exasol/notebook-connector/issues/37)